clone/clone.go

```diff
@@ -485,7 +485,11 @@ func Exec(ctx context.Context, repo, dir string, options ...Option) error {
 	}
 
 	if err := cmd.Start(); err != nil {
-		return err
+		return  err
```

Comment:
```suggestion
		return err
```

Reply:
interesting , i don't have this extra space in my local lol 
![image](https://user-images.githubusercontent.com/43893061/194893895-eec875f9-9955-49a8-a876-2d770944b12c.png)